src/homework/three/Main.java

```diff
@@ -0,0 +1,47 @@
+package homework.three;
+
+import homework.three.phonebook.PhoneBook;
+
+import java.util.*;
+
+public class Main {
+
+   public static void main(String[] args) {
+
+      //      Задание 1.
+      String[] arrayWords = {"whisky", "cognac", "vodka", "whisky", "liquor", "cognac", "whisky", "tequila", "wine", "beer", "wine"};
+      Set<String> uniqueWords = new HashSet<>(Arrays.asList(arrayWords));
+      Map<String, Integer> mapWords = new HashMap<>();
+      for (String uniqueWord : uniqueWords) {
+         int count = 0;
+         for (String word : arrayWords) {
+            if (word.equals(uniqueWord)) {
+               count++;
+            }
+         }
+         mapWords.put(uniqueWord, count);
+      }
```

Comment:
можно совместить в рамках одного цикла
```suggestion
      for (String word : arrayWords) {
         int count = mapWords.getOrDefault(word, 0);
         mapWords.put(word, count + 1);
      }
```